src/main/java/net/hypixel/nerdbot/command/UserCommands.java

```diff
@@ -198,4 +217,253 @@ public static Date parseCustomFormat(String time) throws DateTimeParseException
+            .map(forumId -> NerdBotApp.getBot().getJDA().getForumChannelById(forumId))
+            .filter(Objects::nonNull)
+            .flatMap(forumChannel -> forumChannel.getThreadChannels()
+                .stream()
+                .sorted((o1, o2) -> Long.compare(o2.getTimeCreated().toInstant().toEpochMilli(), o1.getTimeCreated().toInstant().toEpochMilli())) // Sort by most recent
+                .filter(thread -> member == null || thread.getOwnerIdLong() == member.getIdLong())
+                .filter(thread -> thread.getHistoryFromBeginning(1).complete().getRetrievedHistory().get(0) != null) // Lol
+                .filter(thread -> searchTags.isEmpty() || searchTags.stream().allMatch(tag -> thread.getAppliedTags()
+                    .stream()
+                    .anyMatch(forumTag -> forumTag.getName().equalsIgnoreCase(tag))
+                ))
+                .filter(thread -> title == null || thread.getName()
+                    .toLowerCase()
+                    .contains(title.toLowerCase())
+                )
+                .map(thread -> new Suggestion(thread, alpha))
+            )
+            .toList();
+    }
+
+    private static EmbedBuilder buildSuggestionsEmbed(List<Suggestion> suggestions, String tags, String title, boolean alpha, int pageNum) {
+        EmojiConfig emojiConfig = NerdBotApp.getBot().getConfig().getEmojiConfig();
+        List<Suggestion> pages = InfoCommands.getPage(suggestions, pageNum, 10);
+        int totalPages = (int) Math.ceil(suggestions.size() / 10.0);
+        List<List<String>> fieldData = new ArrayList<>();
+        double total = suggestions.size();
+        double greenlit = suggestions.stream().filter(Suggestion::isGreenlit).count();
+
+        for (Suggestion suggestion : pages) {
+            String name = suggestion.getThread().getName().replaceAll("`", "");
+            if (name.length() > 50) {
+                name = name.substring(0, 50);
+                name += "...";
+            }
+            String link = "[" + name + "](" + suggestion.getThread().getJumpUrl() + ")" +
+                (suggestion.isGreenlit() ? " <:creative:" + emojiConfig.getGreenlitEmojiId() + ">" : "");
+
+            fieldData.add(Arrays.asList(
+                link,
+                String.valueOf(suggestion.getAgrees()),
+                String.valueOf(suggestion.getDisagrees())
+            ));
+        }
+
+        EmbedBuilder embedBuilder = new EmbedBuilder();
+        embedBuilder.setColor(Color.GREEN)
+            .setTitle("Suggestions")
+            .setDescription(
+                (tags != null ? "- Filtered by tags: `" + tags + "`\n" : "") + (title != null ? "- Filtered by title: `" + title + "`" : "")
+            )
+            .addField(
+                "Total",
+                String.valueOf((int) total),
+                true
+            )
+            .addField(
+                ("<:creative:" + emojiConfig.getGreenlitEmojiId() + ">"),
```

Comment:
Can we change this to use `jda.getEmojiById(id)` instead? I think it should still show the actually emoji